search/intermediate_result.cpp

```diff
@@ -23,7 +23,39 @@
 
 namespace search
 {
-double const kDistSameStreetMeters = 5000.0;
+namespace
+{
+class SkipRegionInfo
+{
+  static size_t const m_count = 2;
```

Comment:
Please, include cstddef and cstdint for size_t and uint32_t respectively.

Reply:
s/m_count/kCount/

Reply:
done